src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,12 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    @Override
+    public List<? super Machine> get() {
```

Comment:
can we return any type of machine here, if the interface is parameterized with Bulldozer?
let's replace `? super Machine` with Bulldozer too
the same with other producers